modules/soe_paragraph_cta_list/templates/cta-list-paragraph.html.twig

```diff
@@ -32,33 +32,33 @@
 
   {{ paragraph_content }}
 
-  <div class="su-cta-list--links">
+  <div class="su-cta-list__links">
     {%- if link_count > 1 -%}
       <div class="col-1">
         {%- for current_item in field_items|slice(0, (col_1_link_count)) -%}
           {% if not formatter_class is empty %}
-            <a href="{{ current_item.uri }}" class="{{ formatter_class }}">{{ current_item.title }}</a>

```

Comment:
Directly accessing the uri will produce urls like `internal:/node/6` which is no good for the browser. Either let the render array do its thing or use twig functions to translate the links/urls to valid paths.

Reply:
Ok, that's good to know.  I was following the pattern here:
https://github.com/SU-SWS/jumpstart_ui/blob/87a4c7250319adc058535c0c50e677866c0ade11/templates/macros/field-links-to-list.twig#L13

I'm going to do a bit of a deeper dive into twig.  it's a language unto itself.

Reply:
Oh interesting. I think that may be due to the fact one is a link field and the other is a menu item. They look to behave differently.